bin/clinvar_jsons/traits_to_zooma_format.py

```diff
@@ -0,0 +1,207 @@
+import argparse
+import gzip
+import itertools
+import json
+from functools import lru_cache
+import sys
+from time import gmtime, strftime
+
+import progressbar
+import requests
+
+from clinvar_jsons_shared_lib import clinvar_jsons, get_traits_from_json, has_allowed_clinical_significance
+
+
+DATE = strftime("%d/%m/%y %H:%M", gmtime())
+
+
+def main(args):
+    with open(args.outfile_path, "wt") as outfile:
+        outfile.write("STUDY\tBIOENTITY\tPROPERTY_TYPE\tPROPERTY_VALUE\tSEMANTIC_TAG\tANNOTATOR\tANNOTATION_DATE\n")
+        line_count = file_len(args.infile_path)
+        bar = progressbar.ProgressBar(max_value=line_count,
+                                      widgets=[progressbar.AdaptiveETA(samples=1000)])
+        is_zooma_mapping_dict = {}
+        for clinvar_json in bar(clinvar_jsons(args.infile_path)):
+            if not has_allowed_clinical_significance(clinvar_json):
+                continue
+            process_clinvar_json(clinvar_json, outfile, args.zooma_host, args.filters,
+                                 is_zooma_mapping_dict)
```

Comment:
`is_zooma_mapping_dict` is not used in this function and seems to be fully populated inside `process_clinvar_json` so, couldn't it be created inside instead of provided as argument?